blocks/api/factory.js

```diff
@@ -142,7 +143,7 @@ export function switchToBlockType( blocks, name ) {
  */
 export function createReusableBlock( type, attributes ) {
 	return {
-		id: uuid(), // Temorary id replaced when the block is saved server side
+		id: +uniqueId(), // Temorary id replaced when the block is saved server side
```

Comment:
Minor: There's nothing in the signature of `uniqueId` that guarantees it returns a numeric value.

Reply:
Our tests will pick this up if lodash ever changes their implementation